.github/workflows/ci.yml

```diff
@@ -1,6 +1,14 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+    paths-ignore:
+      - 'Documentation/**'
+      - '/*.md'
```

Comment:
Path patterns are already anchored to the root, so `*.md` should suffice to only match `.md` files in the root. Matching all `.md` files throughout the repository would've been `**/*.md`.